go/service/chat_local.go

```diff
@@ -22,23 +24,55 @@ import (
 type chatLocalHandler struct {
 	*BaseHandler
 	libkb.Contextified
-	gh    *gregorHandler
-	boxer *chatBoxer
+	gh      *gregorHandler
+	boxer   *chatBoxer
+	udCache *lru.Cache
 
 	// for test only
 	rc chat1.RemoteInterface
 }
 
 // newChatLocalHandler creates a chatLocalHandler.
 func newChatLocalHandler(xp rpc.Transporter, g *libkb.GlobalContext, gh *gregorHandler) *chatLocalHandler {
+	udc, _ := lru.New(10000)
```

Comment:
An `error` that can only be triggered if you pass in a negative number for the size. It didn't seem worth changing the sig on this just for that case (which we know we won't hit).